src/test/datascience/activation.unit.test.ts

```diff
@@ -2,122 +2,120 @@
 // Licensed under the MIT License.
 
 'use strict';
+export const __ = '';
```

Comment:
Why is this test commented out?

Reply:
Simple, if you check the code, we have Notebook Editor stuff in there.

Reply:
Sure but doesn't the test still apply for native notebooks?